src/scripts/main.js

```diff
@@ -1 +1,79 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const firsPromise = new Promise((resolve, reject) => {
+  const firstClick = body.addEventListener('click', () => {
+    resolve();
+  });
+
+  if (!firstClick) {
+    setTimeout(() => {
+      reject(Error);
+    }, 3000);
+  };
```

Comment:
```suggestion
    setTimeout(() => {
      reject(Error);
    }, 3000);
```
There is no sense in this if since addEventListener always returns undefined which is falsy